product_docs/docs/pgd/5/consistency/conflicts/01_types_of_conflict.mdx

```diff
@@ -0,0 +1,403 @@
+---
+title: Types of Conflict
+Description: Details on different kinds of database conflicts.
+deepToC: true
+---
+
+## PRIMARY KEY or UNIQUE conflicts
+
+The most common conflicts are row conflicts, where two operations affect a row with the same key in ways they can't on a single node. PGD can detect most of those and applies the `update_if_newer` conflict resolver.
+
+Row conflicts include:
+
+-   `INSERT` versus `INSERT`
+-   `UPDATE` versus `UPDATE`
+-   `UPDATE` versus `DELETE`
+-   `INSERT` versus `UPDATE`
+-   `INSERT` versus `DELETE`
+-   `DELETE` versus `DELETE`
+
+The view `bdr.node_conflict_resolvers` provides information on how conflict resolution is currently configured for all known conflict types.
+
+### INSERT/INSERT conflicts
+
+The most common conflict, `INSERT`/`INSERT`, arises where `INSERT` operations on two different nodes create a tuple with the same `PRIMARY KEY` values (or if no `PRIMARY KEY` exists, the same values for a single `UNIQUE` constraint).
+
+PGD handles this situation by retaining the most recently inserted tuple of the two according to the originating node's timestamps. (A user-defined conflict handler can override this behavior.)
+
+This conflict generates the `insert_exists` conflict type, which is by default resolved by choosing the newer row, based on commit time, and keeping only that one (`update_if_newer` resolver). You can configure other resolvers. See [Conflict resolution](#conflict-resolution) for details.
```

Comment:
Not sure, but maybe this link needs to go to https://deploy-preview-5804--edb-docs-staging.netlify.app/docs/pgd/latest/consistency/conflicts/04_conflict_resolution/ ? 

```suggestion
This conflict generates the `insert_exists` conflict type, which is by default resolved by choosing the newer row, based on commit time, and keeping only that one (`update_if_newer` resolver). You can configure other resolvers. See [Conflict resolution](04_conflict_resolution) for details.
```

Reply:
fixed